app/models/host_status/execution_status.rb

```diff
@@ -1,17 +1,21 @@
 class HostStatus::ExecutionStatus < HostStatus::Status
+  # execution finished successfully
   OK = 0
```

Comment:
I think these were reasons why host status was implemented with numbers
* compatibility with bitmask field that's used in configuration status (same is used by openscap status)
* column size

Another benefit I see is that if some state changes later because we find better word for it, we don't have to change existing records nor the value we display in API.

If we introduced symbol representation we'd have to provide some mapper, since our ExecutionStatus inherits from core HostStatus::Status object. I don't thing it's worth it.